File: testprojects/python_main/pipelines/PERF_UNITEST_GENERATE/code/perf_unitest_generate/graph/Deduplicate_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from perf_unitest_generate.config.ConfigStore import *
from perf_unitest_generate.udfs.UDFs import *

def Deduplicate_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.dropDuplicates(["account_open_date"])

File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/Deduplicate_1.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import config.Context
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Deduplicate_1 {

  def apply(context: Context, in: DataFrame): DataFrame = {
    import org.apache.spark.sql.expressions.Window
    in.dropDuplicates(
      List("c  - int", "- c long", "c_decimal  -  ", "c_float-__  ")
    )
  }

}
